### PR TITLE
[regression](decimalv3)Add decimal type as filter condition in regression test

### PR DESCRIPTION
**Pull request by caoliang-web**
## Problem summary

1. Add decimal type as filter condition in regression test

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/decimalv3/test_decimalv3_where.groovy [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+suite("test_decimalv3_where") {
+
+
+    def tableName = "test_decimalv3_demo"
+	sql """drop table if exists ${tableName}"""
+	sql """CREATE TABLE ${tableName} ( `id` varchar(11) NULL COMMENT '唯一标识', `name` varchar(10) NULL COMMENT '采集时间', `age` int(11) NULL, `dr` decimalv3(10, 0) ) ENGINE=OLAP UNIQUE KEY(`id`) COMMENT 'test' DISTRIBUTED BY HASH(`id`) BUCKETS 10 PROPERTIES ( "replication_allocation" = "tag.location.default: 1", "in_memory" = "false", "storage_format" = "V2", "light_schema_change" = "true", "disable_auto_compaction" = "false" );"""
+	sql """insert into ${tableName} values (1,'doris',20,324.10),(2,'spark',10,95.5),(3,'flink',9,20)"""
+	qt_decimalv3 "select * from ${tableName} where dr != 1 ;"
+}
```
Comment (yiguolei) [inline regression-test/suites/decimalv3/test_decimalv3_where.groovy:26]: should add order by to make test stable.

**Comment by yiguolei** [comment]
./run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by yiguolei** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**